src/rabbit_mgmt_db_handler.erl

```diff
@@ -42,18 +42,34 @@ add_handler() ->
 gc() ->
     erlang:garbage_collect(whereis(rabbit_event)).
 
+%% application:get_env/3 is only available in R16B01 or later
+get_env(Application, Key, Def) ->
+    case application:get_env(Application, Key) of
+    {ok, Val} ->
+        Val;
+    undefined ->
+        Def
+    end.
```

Comment:
Could you move this function to `rabbit_misc:get_app_env/3` or `app_utils:get_env/3` in the broker for instance? It could be useful in several places.